Existing set of patches from the desktop gate for Solaris
Not suitable for upstream

--- libxklavier-5.0.orig/libxklavier/xklavier_config.c	2009-06-24 13:08:38.000000000 +0200
+++ libxklavier-5.0/libxklavier/xklavier_config.c	2012-06-05 14:10:30.908824990 +0200
@@ -1049,6 +1049,7 @@
 	object_class->set_property = xkl_config_registry_set_property;
 	object_class->get_property = xkl_config_registry_get_property;
 
+	bindtextdomain(XKB_DOMAIN, LOCALEDIR);
 	bind_textdomain_codeset(XKB_DOMAIN, "UTF-8");
 
 	engine_param_spec = g_param_spec_object("engine",

